corehq/apps/app_manager/tests/test_profile.py

```diff
@@ -85,7 +85,7 @@ def _test_property(self, profile, key, value, setting):
 
     def _test_dependencies(self, profile, key, value, setting):
         node = profile.find('./features/dependencies')
-        if node:
+        if node is not None:
```

Comment:
Same question about how you found these instances. Failing tests?

Reply:
Yes, failing tests like [this one](https://github.com/dimagi/commcare-hq/actions/runs/14228464664/job/39873565123), and then grepping for things that looked like XML nodes being evaluated in a boolean context. I may not have gotten all of them, but since it's a deprecation warning it will have no effect in production. New tests are added that trigger the warning will fail (which is good).